apps/dh/e2e-dh/src/integration/b2c.spec.ts

```diff
@@ -38,6 +37,10 @@ const environments = [
     name: 'B002',
     url: 'https://purple-forest-07e41fb03.azurestaticapps.net',
   },
+  {
+    name: 'itlev',
+    url: 'https://itlev.datahub.dk',
+  },
```

Comment:
question: Not sure if this is a valid concern but right now anyone can find these URLs on GitHub and create a user. Then they can do anything in the app, including changing names, triggering actions, reading information. Especially with the ITLev environment, isn't that risky?

Reply:
I see your point, however everything should go through APIM so they shouldn't be able to do stuff they aren't allowed to. Anyways better safe than sorry, so I will remove it for now.